Make sure that frozenlists are pickle-able (#718)
aio-libs/frozenlist: CHANGES/718.bugfix.rst

> Fix bug where FrozenList could not be pickled/unpickled.

Reviewer: This would be a bit better but ideally, it should explain the effect the end-users would see in an observable way.
```suggestion
Fixed a bug where FrozenList could not be pickled/unpickled
```

Author: I'll rewrite this to make more sense.

> Fixed a bug where FrozenList could not be pickled/unpickled.

Reviewer: ```suggestion
Fixed a bug where :class:`~frozenlist.FrozenList` could not be pickled/unpickled.
```

> Pickling is a requirement for FrozenList to be able to be passed to multiprocessing Processes. Without this

Reviewer: ```suggestion
Pickling is a requirement for :class:`~frozenlist.FrozenList` to be able to be passed to :class:`multiprocessing.Process`\ es. Without this
```

> fix, users receive a TypeError upon attempting to pickle a FrozenList.

Reviewer: ```suggestion
fix, users receive a :exc:`TypeError` upon attempting to pickle a :class:`~frozenlist.FrozenList`.
```

> -- by :user:`csm10495`.

Reviewer: To make the byline detached, put it on a dedicated line and drop the trailing period.
```suggestion

-- by :user:`csm10495`
```
